content/sensu-enterprise/3.2/guides/troubleshooting.md

```diff
@@ -0,0 +1,103 @@
+- [Settings Log Levels](#setting-log-levels)
+- [Printing Configuration](#printing-configuration)
+- [Resolving Private Key Errors](#resolving-private-key-erors)
+
+### Reloading Configuration
+
+Following any changes to the Sensu Enterprise configuration, you must reload Sensu Enterprise in order for the changes to take effect.
+
+Reload Sensu Enterprise on Linux distributions using `systemd`:
+
+{{< highlight shell >}}
+sudo systemctl reload sensu-enterprise{{< /highlight >}}
+
+Reload Sensu Enterprise on Linux distributions using `sysvinit`:
+
+{{< highlight shell >}}
+sudo service sensu-enterprise reload{{< /highlight >}}
+
+When making changes to the Sensu client configuration, restart the client using the process described in the [Sensu Core troubleshooting guide][2].
+
+### Collecting Logs
+
+Sensu's logs provide a wealth of information when troubleshooting issues. They live at `/var/log/sensu`:
+
+{{< highlight shell >}}
+/var/log/sensu
+├── sensu-enterprise.log
+└── sensu-enterprise-dashboard.log{{< /highlight >}}
+
+You can view the Sensu Enterprise logs at the paths above, or provide them in an archive:
+
+{{< highlight shell >}}
+tail -n 10000 /var/log/sensu/sensu-enterprise.log > sensu-enterprise-10k.log && gzip -9 sensu-enterprise-10k.log{{< /highlight >}}
+
+### Setting Log Levels
+
+Sensu has the ability to set log levels interactively, or by using a configuration directive in `/etc/default/sensu-enterprise`. This is particularly useful when attempting to debug an issue where the current log level doesn't provide sufficient information. Let's take a look at the ways you can set your log levels.
+
+The quickest way to toggle the `debug` log level on/off for Sensu Enterprise is to use `sudo kill -TRAP $SENSUPID`:
+
+{{< highlight shell >}}
+sudo ps aux | grep [s]ensu-enterprise
+sensu     5992  1.7  0.3 177232 24352 ...
+sudo kill -TRAP 5992{{< /highlight >}}
+
+Additionally, you can set the log level to `info` or `debug` by using the configuration directive in `/etc/default/sensu-enterprise`. Let's take a look at an example:
+
+{{< highlight shell >}}
+sudo cat /etc/default/sensu-enterprise
+LOG_LEVEL=debug{{< /highlight >}}
+
+And after setting that directive, [reload Sensu Enterprise](#reloading-configuration):
+
+{{< highlight shell >}}
+sudo systemctl reload sensu-enterprise{{< /highlight >}}
+
+Keep in mind that to set log levels back to normal, you can either run `sudo kill -TRAP $SENSUPID` (if you've used that method), or revert the change in `/etc/default/sensu-enterprise` and reload Sensu Enterprise for the change to take place.
+
+_NOTE: By default, Sensu's logging level is set to `info`. However, there are more log levels available than just `info` and `debug`. You can find the full list of available log levels in the [configuration reference documentation][4]. It's worth noting that `debug` is the most granular log level, while `fatal` is the least granular._
+
+### Printing Configurations
```

Comment:
It is worth a note somewhere in here I think to not run that print config in `/etc/sensu/conf.d/`, as Sensu will pickup that as a configuration file as it ends in `.json` on the next reload or restart, causing some fun issues.

I'd suggest `cd`ing to `/tmp` first or just doing `tee /tmp/se-config.json` on the end instead.

The command also outputs it to `stdout`, so we may also want to call out that this file gets created, as some users may not even realize it was done at all and just copy `stdout`.

Reply:
Fixed in https://github.com/sensu/sensu-docs/pull/790/commits/6ed9b0d7f943a2064ddaa89891e48531b59977f8